diff --git a/setup.py b/setup.py
index d4ccdb7..dc65b49 100644
--- a/setup.py
+++ b/setup.py
@@ -69,23 +69,19 @@ if _ABLE_TO_COMPILE_EXTENSIONS:
 
         # list of libraries that will be bundled with python connector,
         # this list should be carefully examined when pyarrow lib is
-        # upgraded
+        # upgraded. For conda, we don't vendor at all
         arrow_libs_to_copy = {
-            "linux": ["libarrow.so.800", "libarrow_python.so.800", "libparquet.so.800"],
-            "darwin": [
-                "libarrow.800.dylib",
-                "libarrow_python.800.dylib",
-                "libparquet.800.dylib",
-            ],
-            "win32": ["arrow.dll", "arrow_python.dll", "parquet.dll"],
+            "linux": [],
+            "darwin": [],
+            "win32": [],
         }
 
         arrow_libs_to_link = {
-            "linux": ["libarrow.so.800", "libarrow_python.so.800", "libparquet.so.800"],
+            "linux": ["libarrow.so", "libarrow_python.so", "libparquet.so"],
             "darwin": [
-                "libarrow.800.dylib",
-                "libarrow_python.800.dylib",
-                "libparquet.800.dylib",
+                "libarrow.dylib",
+                "libarrow_python.dylib",
+                "libparquet.dylib",
             ],
             "win32": ["arrow.lib", "arrow_python.lib", "parquet.lib"],
         }
