From 1412a7d6ab948fe79d4f34c3a016a6d5f3604f41 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Fri, 27 May 2022 12:22:49 +0000
Subject: [PATCH] Fix pyarrow in setup.py

---
 setup.py  | 8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index 9244cab..3b490b7 100644
--- a/setup.py
+++ b/setup.py
@@ -69,14 +69,14 @@ if _ABLE_TO_COMPILE_EXTENSIONS:
         # this list should be carefully examined when pyarrow lib is
         # upgraded
         arrow_libs_to_copy = {
-            "linux": ["libarrow.so.600", "libarrow_python.so.600"],
-            "darwin": ["libarrow.600.dylib", "libarrow_python.600.dylib"],
+            "linux": ["libarrow.so", "libarrow_python.so"],
+            "darwin": ["libarrow.dylib", "libarrow_python.dylib"],
             "win32": ["arrow.dll", "arrow_python.dll"],
         }
 
         arrow_libs_to_link = {
-            "linux": ["libarrow.so.600", "libarrow_python.so.600"],
-            "darwin": ["libarrow.600.dylib", "libarrow_python.600.dylib"],
+            "linux": ["libarrow.so", "libarrow_python.so"],
+            "darwin": ["libarrow.dylib", "libarrow_python.dylib"],
             "win32": ["arrow.lib", "arrow_python.lib"],
         }
 
-- 
2.34.1

